examples/third_party/BUILD.bazel

```diff
@@ -31,6 +31,7 @@ test_suite(
         "//libssh2:libssh2_build_test",
         "//openssl:openssl_build_test",
         "//pcre:pcre_build_test",
+        "//python:python_test",
```

Comment:
Could you also add this to other platforms? It'd be ideal if this could be added to all test_suites.

Reply:
I'm happy to add it to macos, although i do not own a mac so cannot resolve any issues. I'll add the test to the macos test suite and see what happens

Reply:
Is there any reason this wouldn't be expected to work on windows as well?

Reply:
I suppose not, although there aren’t any tests at all for Windows at the moment. Happy to add it to the test suite to give it a go!

Reply:
I'm guessing on windows you'll run into problems with compiling openssl and also the `.exe` extensions on the binaries will need some select statements adding.

Reply:
Ah yeah, the OpenSSL build doesn't work :(